Automattic/wp-calypso
# Data Stores: Remove unused i18n store

## Proposed Changes

This PR removes the `i18n` store that we are no longer using since #74475.

## Testing Instructions

* Verify the removed store is no longer in use.
* Verify all checks are green.
